includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,271 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $data = [
+		"logo" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "http://fandom.wikia.com",
+				"image" => "wds/full_fandom_logo",
+				"title" => [
+					"type" => "text",
+					"value" => "Fandom",
+				]
+			]
+		],
+		"verticals" => [
+			"links" => [
+				[
+					"type" => "link-branded",
+					"brand" => "tv",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-tv",
+					],
+					"href" => "http://tv.wikia.com"
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "games",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-games",
+					],
+					"href" => "http://games.wikia.com",
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "movies",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-movies",
+					],
+					"href" => "http://movies.wikia.com",
+				],
+			],
+		],
+		"wikis" => [
+			"header" => [
+				"type" => "link-branded",
+				"brand" => "wikis",
+				"href" => "#",
+				"title" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-brandlink-vertical-wikis",
+				],
+			],
+			"links" => [
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-explorewikis",
+					],
+					"href" => "#",
+				],
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-wikis-communitycentral",
+					],
+					"href" => "#",
+				],
+			],
+		],
+		"search" => [
+			"module" => [
+				"type" => "search",
+				"results" => [
+					"url" => "http://wikia.com/search",
+					"param" => "query",
```

Comment:
We are using `param` in many places here (ie. https://github.com/Wikia/app/pull/11196/files#diff-4a1b823158dfc4984cc02216f2910d32R115). Do you think we should change all occurences?